config/tablet/default.yaml

```diff
@@ -98,7 +98,7 @@ hotRowProtection:
   maxGlobalQueueSize: 1000    # hot_row_protection_max_global_queue_size
   maxConcurrency: 5           # hot_row_protection_concurrent_transactions
 
-consolidator: enable|disable|notOnMaster # enable-consolidator, enable-consolidator-replicas
+consolidator: enable|disable|notOnPrimary # enable-consolidator, enable-consolidator-replicas
```

Comment:
Is this a breaking change? Users could have specified `notOnMaster` in a config somewhere?

Reply:
That will still work. We are deprecating the old value but will still respect it if provided.
https://github.com/vitessio/vitess/blob/main/go/vt/vttablet/tabletserver/tabletserver.go#L1817-L1819